packages/sawja/sawja.1.5.3/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "sawja@inria.fr"
+build: [
+  ["./configure.sh"]
+  [make]
+]
+remove: [
+  ["ocamlfind" "remove" "sawja"]
+]
+bug-reports: "sawja@inria.fr"
+author: "Sawja development team"
+install: [
+  [make "install"]
+]
+homepage: "http://sawja.inria.fr"
+available: ocaml-version >= "4.02" 
+depends: [
+  "ocamlfind"
```

Comment:
Could you add the ```{build}``` attribute here unless you are using the ```Findlib``` library ? This avoids compiling everything everytime ocamlfind gets upgraded/rebuilt.

Reply:
Thanks for your help.
I'm not sure to understand. We are using findlib. 
Which line should I modify? How?

Reply:
Mmh looking at your code it doesn't seems to be the case. You are using the ocamlfind program but not the ```Findlib``` library (shipped in the same packages).
Unless I'm missing something this line should probably be ```"ocamlfind" {build}``` instead.

For more info you can look at the "Build depends" section in the documentation: https://opam.ocaml.org/doc/Packaging.html#More-on-opam-files

Reply:
Done.